dev-python/sortedcontainers/sortedcontainers-2.1.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Python library to sort collections and containers"
+HOMEPAGE="http://www.grantjenks.com/docs/sortedcontainers/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Why drop the existing `~` arches here? It doesn't seem necessary.

Reply:
I dropped them because I can't test the package on these architectures, and as I understood it, I shouldn't include architectures I can't test on. But I can re-add them I suppose.

Reply:
For new packages: yes! For this case with a bump — assuming no known breakages — I believe it's safe to keep in. Happy to learn otherwise.